_posts/2017-09-05-Browser Security Features.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+authors: [tim_de_grande]
+title: "Browser Security Features"
+image: /img/security/padlock_code.jpg
+tags: [Security]
+category: Security
+comments: true
+permalink: conference/2017/09/05/Browser-security-features.html
+---
+# Browser security features
+Browsers nowadays come with a ton of security features built-in.
+These features are there to protect the people using your application as well as protecting your application from malicious others.
+Most of these features are quite easy to implement, however for some of them (such as key-pinning), you have to be careful not to break your site.
```

Comment:
Just for nitpicking: the comma after "(such as key-pinning)" shouldn't be there 🙃